### ISSUE TITLE
Check if session shall be started

### ISSUE THREAD

**Issue by far4599** [SocialConnect/auth#29]
Start session if session does not exist:

> src/Provider/Session/Session.php

```php
namespace SocialConnect\Provider\Session;

class Session implements SessionInterface
{
    public function __construct()
    {
        if (session_status() == PHP_SESSION_NONE) { // New line
            session_start();
        } // New line
    }

```

**Comment by ovr** [comment]
Hey!

Thank you for bug reporting, I am going to prepare patch and new release!

**Comment by ovr** [comment]
I submitted patches to master (`1.1` branch - it's new upcoming release - `dev-master`)
And I released 1.0.1 (patch for `1.0` release)

Thanks
